gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/tasks/LaunchConfigTask.java

```diff
@@ -170,7 +167,8 @@ public final void createConfig() throws IOException {
                         .serviceName(serviceName.get())
                         .javaHome(javaHome.getOrElse(""))
                         .args(args.get())
-                        .classpath(relativizeToServiceLibDirectory(classpath))
+                        .classpath(relativizeToServiceLibDirectory(getClasspath()))
+                        .addAllJvmOpts(javaAgentArgs())
```

Comment:
Just to check, because I'm not sure, we had a problem with deployments overriding particular jvm args in the past, these here are in a place where they won't get overridden, right (as we want agents - or at least your new agent - to always be applied)?

Reply:
Yep, much like the classpath I don't want to allow users to modify agents after build time. If it's included in the distribution we should be able to rely on it always executing.